sequencer/api/availability.toml

```diff
@@ -58,3 +58,22 @@ where a `Header` has the schema:
 
 All timestamps are denominated in an integer number of seconds.
 """
+
+[route.getheader]
+PATH = ["header/:height"]
+":height" = "Integer"
+DOC = """
+If the header at the provided height is available, returns
+
+```json
+{
```

Comment:
I guess this is copied from above but this format is actually out of data. The header now looks like

```json
{
  "metadata": {
    "timestamp": "integer",
    "l1_head": "integer",
    "l1_finalized": { ... },
  },
  "transactions_root": { ... },
}
```